### ISSUE TITLE
Final Results

### ISSUE THREAD

**Issue by Miaohua9** [tntek/source-free-domain-adaptation#4]
Thanks for your great work and sharing. 

I noticed that in some paper, the data_list is arranged in order, 1,2,3, and in your data_list like Art_list.txt, it is a random order. 
When you report the results in the table, do you use the same style list file for all the methods? I notice that these two style list can produce difference results with a gap. 

Did the reported results in the paper  use the accuracy from the final Epoch or the Best Epoch? I noticed that sometimes the final epoch is not the best one.

Thank you very much.

**Comment by HazelSu** [comment]
q1: "DataLoader(dsets["target"], batch_size=train_bs, shuffle=True, num_workers=cfg.NUM_WORKERS, drop_last=False)" during the training, dataloader has already shuffled.
q2:the final
